pages/angular.md

```diff
@@ -9,35 +9,24 @@ Create a prototype Progressive Web App using Angular & CouchDB with the BeLL App
     1. Writing front end code
     2. Helping with unit tests
     3. Helping with e2e tests
-2. Help with broader backend goals: SSL, Service Workers,
-
+2. Making this project "Offline First"
+3. Making the app secure with SSL
 
 ## Installation
 
-1. **Clone** the [GitHub repository](https://github.com/ole-vi/BeLL-angular-reboot)
-    *Note: This is in the process of switching to a [different repository](https://github.com/ole-vi/planet)*
-2. If you haven't already, install [CouchDB 2.0](http://couchdb.apache.org/) and [NodeJS](https://nodejs.org/en/)
-3. In a bash terminal, run the following commands to setup your CouchDB main databases:
-
-    ```
-    curl -X PUT http://127.0.0.1:5984/_users
-    curl -X PUT http://127.0.0.1:5984/_replicator
-    curl -X PUT http://127.0.0.1:5984/_global_changes
-
-    ```
-
-4. Then go into the CouchDB dashboard at `http://127.0.0.1:5984/_utils` and enable CORS by clicking __Configuration__ -> __CORS__, click the __Enable CORS__ button and add `http://localhost:3000` and/or `http://127.0.0.1:3000` to the list
- 
-    *Paul's note: add image here*
+1. You should have Vagrant installed already, but if not install Vagrant
```

Comment:
could we also add the link for vagrant.
And yes virtual box as well something similar to BeLL-Apps documentation.
And one issue that happens with virtual box is that it needs to run on VT-x not just parallelism